submissions/VBystrov/pre-oop/index.js

```diff
@@ -0,0 +1,96 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/VBystrov/a-tiny-JS-world
+   Web app: https://vbystrov.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+const dog = {
+  species: 'dog',
+  name: 'Globgor',
+  gender: 'male',
+  legs: 4,
+  hands: 0,
+  friends: ['Eclipsa'],
+  saying: 'woof!',
+};
+
+const cat = {
+  species: 'cat',
+  name: 'Eclipsa',
+  gender: 'female',
+  legs: 4,
+  hands: 0,
+  friends: ['Globgor', 'Star'],
+  saying: 'mew-mew-mew-mew',
+};
+
+const woman = {
+  species: 'homo sapiens',
+  name: 'Star',
+  gender: 'female',
+  legs: 2,
+  hands: 2,
+  friends: ['Eclipsa', 'Marko'],
+  saying: "I'm a magical princess from another dimension!",
+};
+
+const man = {
+  species: 'homo sapiens',
+  name: 'Marko',
+  gender: 'male',
+  legs: 2,
+  hands: 2,
+  friends: ['Star'],
+  saying: "I'm a misunderstood bad boy!",
+};
+
+const catWoman = {
+  species: 'homo catus',
+  name: 'Amelia',
+  gender: 'female',
+  legs: 2,
+  hands: 2,
+  friends: [],
```

Comment:
Maybe we need to add some friends to this list as well or make and additional check inside the `getTemplate` function

Reply:
Added check for arrays.